### ISSUE TITLE
Anyway to check the internal dictionary. a word is misspelled but not detected

### ISSUE THREAD

**Issue by qiansen1386** [streetsidesoftware/vscode-spell-checker#2655]
I misspell `retrieve` to `retrive`, but it is not flagged out? Anyway to check how to prevent it from happening?

**Comment by Jason3S** [comment]
@qiansen1386,

The spell checker uses `cspell` to do the spell checking. It has a `trace` command.

`cspell trace retrive`
<img width="821" alt="image" src="https://github.com/streetsidesoftware/vscode-spell-checker/assets/3740137/47e62841-07a7-4e5f-bf98-e5264daed69d">

`cspell trace --allow-compound-words retrive`
<img width="830" alt="image" src="https://github.com/streetsidesoftware/vscode-spell-checker/assets/3740137/41d0b9a5-cc41-48af-a076-ffcee8962b05">

I suggest turning off `allowCompoundWords`.

## Forbidding `retrive`

The `cSpell.flagWords` setting can help with forbidding it.

<img width="705" alt="image" src="https://github.com/streetsidesoftware/vscode-spell-checker/assets/3740137/b0cb3049-3fcc-4a26-976e-420ef3918711">



## Installing `cspell`

```
npm install -g cspell
```

**Comment by qiansen1386** [comment]
I see. Thanks so much for the detailed explanation.
